packages/wallets/src/components/Base/WalletText/WalletText.tsx

```diff
@@ -1,33 +1,33 @@
 import React, { ComponentProps, CSSProperties, ReactNode } from 'react';
 import classNames from 'classnames';
 import { TGenericSizes } from '../types';
-import styles from './WalletText.module.css';
+import './WalletText.scss';
 
 interface WalletTextProps extends ComponentProps<'span'> {
     align?: CSSProperties['textAlign'];
     children: ReactNode;
     color?: CSSProperties['color'] | 'error' | 'general' | 'primary' | 'success' | 'warning';
     lineHeight?: TGenericSizes;
-    size?: TGenericSizes;
+    size?: Exclude<TGenericSizes, '5xl' | '6xl'>;
```

Comment:
just a question also, why is the `5xl` and `6xl` excluded thou?

Reply:
I am excluding them because they are not an option for size. I feel I should update the TGenericSizes aswell, because some of the values I am not using aswell.